action/builder_test.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2018 IoTeX
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package action
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/iotexproject/iotex-core/pkg/version"
+	"github.com/iotexproject/iotex-core/test/testaddress"
+)
+
+func TestActionBuilder(t *testing.T) {
+	srcAddr := testaddress.Addrinfo["producer"]
+	dstAddr := testaddress.Addrinfo["echo"]
+	bd := &Builder{}
+	act := bd.SetVersion(version.ProtocolVersion).
+		SetNonce(2).
+		SetSourceAddress(srcAddr.RawAddress).
+		SetSourcePublicKey(srcAddr.PublicKey).
+		SetDestinationAddress(dstAddr.RawAddress).
+		SetGasLimit(10003).
+		SetGasPrice(big.NewInt(10004)).
+		Build()
+
+	assert.Equal(t, uint32(version.ProtocolVersion), act.Version())
```

Comment:
change to require? we agreed to use require, existing assert should be changed to require when we modify existing codes.